packages/ps2/src/apis/user/use.tsx

```diff
@@ -148,7 +149,13 @@ export const useAuthenticate = (): [
 
 export function useLogout(): () => void {
   const dispatch = useDispatch();
+  const [logoutRequest] = useLogoutMutation();
   return () => {
+    try {
+      logoutRequest();
+    } catch (e) {
+      throw e;
+    }
```

Comment:
```suggestion
    logoutRequest();
```


@Nikita9901 in the original code you've shown me, it made sense because we had an await. We do not not need the await here because we... well, we just don't need to wait for it or make sure the request goes successful.
So you did right removing the await.

But now the try catch makes no sense because it will never catch anything since, even if we get a backend error, we will not catch it since we do not await that promise.